components/server/src/bitbucket-server/bitbucket-server-repository-provider.ts

```diff
@@ -194,7 +194,7 @@ export class BitbucketServerRepositoryProvider implements RepositoryProvider {
         });
 
         const commits = commitsResult.values || [];
-        return commits.map((c) => c.id);
+        return commits.map((c) => c.id).slice(1);
```

Comment:
This is the place we change behavior. AFAIK, it did not cause any issues. This is because when we had checks checking things about each historical revision (like if there exists a prebuild for it), we just failed to find a prebuild twice instead of only once.

This way, the only impacted feature was our configurable prebuilds interval, which must've been off by one this whole time.

Reply:
Nice find! :heart: